tests/wpt/tests/css/CSS2/tables/table-vertical-align-baseline-008.xht

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml">
+ <head>
+  <title>CSS Test: Test for baseline alignment of table cells</title>
+  <link rel="author" title="Oriol Brufau" href="obrufau@igalia.com" />
+  <link rel="help" href="https://github.com/servo/servo/issues/31722" />
+  <link rel="help" href="https://drafts.csswg.org/css2/#height-layout" />
+  <link rel="match" href="../reference/ref-filled-green-100px-square.xht" />
+  <meta name="assert" content="Since the cell is empty, its baseline is at the bottom." />
```

Comment:
It might be nice to say the bottom of what here -- the row or the table, etc. Or maybe just say that empty cells don't set a baseline.